README.md

```diff
@@ -198,3 +198,25 @@ From the _root_ directory of the repository:
 ```shell
 pytest tests/
 ```
+
+### Docker Setup
+
+From the root directory , where Docker and docker-compose file are , run the following commands:  
+
+*docker-compose up -d db
```

Comment:
I think it is clearer to rename `db` to `seqrepo`

Reply:
Agree, your team can finalise the naming convention for the services . I will rename them accordingly.